lambdas/issue-client-access-token/src/main/java/uk/gov/di/ipv/core/issueclientaccesstoken/validation/TokenRequestValidator.java

```diff
@@ -72,7 +72,7 @@ private void validateMaxAllowedAuthClientTtl(JWTAuthenticationClaimsSet claimsSe
         }
     }
 
-    private void validateJwtId(JWTAuthenticationClaimsSet claimsSet) {
+    private void validateJwtId(JWTAuthenticationClaimsSet claimsSet) throws InvalidClientException {
         JWTID jwtId = claimsSet.getJWTID();
         if (jwtId == null || StringUtils.isBlank(jwtId.getValue())) {
             LOGGER.warn(LogHelper.buildLogMessage("The client auth JWT id (jti) is missing"));
```

Comment:
Do you think we can turn this into an error now as well? (Otherwise you can bypass the replay detection by omitting a JTI!)

I can't see any instances of this log message in prod, but we should be a little careful that it doesn't break anything!

Reply:
ack good point - thanks!

Reply:
No worries - it wasn't on the ticket :)